daprdocs/content/en/getting-started/quickstarts/serviceinvocation-quickstart.md

```diff
@@ -205,17 +203,22 @@ Install the dependencies:
 npm install
 ```
 
-Run the `order-processor` service alongside a Dapr sidecar.
+Run the `checkout` service alongside a Dapr sidecar.
 
 ```bash
-dapr run --app-port 6001 --app-id order-processor --app-protocol http --dapr-http-port 3501 -- npm start
+dapr run  --app-id checkout --app-protocol http --dapr-http-port 3500 -- npm start
```

Comment:
```suggestion
dapr run --app-id checkout --app-protocol http --dapr-http-port 3500 -- npm start
```

Reply:
I've never put two spaces after dapr run, is that some kind of style guideline ?

Reply:
Nope - was there before my PR. Just checked v1.6 branch